frontend/src/components/index.ts

```diff
@@ -5,5 +5,6 @@ import { NavBar } from "./NavBar.tsx";
 import { Page } from "./Page.tsx";
 import { Pathway } from "./Pathway.tsx";
 import { PathwayTimeline } from "./PathwayTimeline.tsx";
+import { PrescreeningForm } from "./PrescreeningForm.tsx";
```

Comment:
This import should be in the `index.ts` file in the pages folder after you moved the files